src/GitHub.Exports/Models/CommentModel.cs

```diff
@@ -11,6 +11,16 @@ public class CommentModel
         /// Gets the ID of the comment.
         /// </summary>
         public string Id { get; set; }
+        
+        /// <summary>
+        /// Gets the DatabaseId of the comment.
+        /// </summary>
+        public int DatabaseId { get; set; }
+
+        /// <summary>
+        /// Gets the PullRequestId of the comment
+        /// </summary>
+        public int PullRequestId { get; set; }
```

Comment:
Yeah, it's not ideal having this here, but because the REST API requires it, there's not really a better way. Hopefully we'll be able to delete comments using GQL soon!

Reply:
Might want to add a comment saying that this is just needed for deleting comments via the REST API and should be temporary?

Reply:
Updated